gpbft/api.go

```diff
@@ -54,9 +54,11 @@ type MessageReceiver interface {
 
 // Interface which network participants must implement.
 type Receiver interface {
-	// Begins executing the protocol.
+	// Beings the execution of the instance number passed as an argument.
 	// The node will request the canonical chain to propose from the host.
-	Start() error
```

Comment:
Please make sure PR description covers the rationale for this change since it seems unrelated to the purpose of this PR?

Reply:
I elaborated a bit more the description. Let me know if it is clear now. Thanks!